docker/build-dcos-ui

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Switch to branch or commit hash
+
+if [[ ! -z "$PLUGINS_CHECKOUT" && -d "./dcos-ui-plugins" ]]; then
+  cd ./dcos-ui-plugins
+  git pull
+  git checkout $PLUGINS_CHECKOUT
+fi
+
+cd /usr/src/dcos-ui
+git pull
+git checkout $APPLICATION_CHECKOUT
```

Comment:
But then, if the user misspelled `APPLICATION_CHECKOUT` and didn't want master, they would be confused. I much rather this just fail and they have to fix.
Do you think having a default adds much value?